server/src/graphql/api.ts

```diff
@@ -32,10 +32,21 @@ export const graphqlRoot: Resolvers<Context> = {
       return thing
       // return arts.find(art => art.id === id) || null
     },
-    arts: async () => {
-      const thing = await Art.find()
-      console.log(thing)
-      return thing
+    arts: async (_, { checkSeen }, ctx) => {
+      // BUG: Change this to use art object seen by.
+      // Blocked on decoupling type systems
```

Comment:
So right now if I tried to call this query with checkSeen as true will it loop between arts and users indefinitely?

Reply:
No, it will just be slightly more inefficient than I would like